composer.json

```diff
@@ -165,7 +163,6 @@
         "toflar/cronjob-supervisor": "^2.0",
         "toflar/psr6-symfony-http-cache-store": "^4.0",
         "twig/extra-bundle": "^3.0",
-        "twig/string-extra": "^3.0",
```

Comment:
@m-vo It seems we no longer use this package, do we?

Reply:
See https://github.com/contao/contao/pull/7104#discussion_r1559334992 _(I missed this comment before)_

Reply:
Added again in 9aab35c176db5be0dd960f056ded4984103edaec.